src/drivers/Cst816s.cpp

```diff
@@ -32,6 +32,18 @@ bool Cst816S::Init() {
   twiMaster.Read(twiAddress, 0xa7, &dummy, 1);
   vTaskDelay(5);
 
+  static constexpr uint8_t maxRetries = 3;
+  bool isDeviceOk = false;
+  uint8_t retries = 0;
+  do {
+    isDeviceOk = CheckDeviceIds();
+    retries++;
+  } while(!isDeviceOk && retries < maxRetries);
```

Comment:
It's not really a huge issue, but is this style the one we want to use?

Reply:
Do you mean `!` vs `not` and `&&` vs `and` ?

Reply:
`do {} while()` instead of `while () {`

Reply:
To me, the `do while` loop makes sense as I want to ensure the check is done at least once. But I agree the initialization of `isDeviceOk` is probably redundant in this case...